lib/fixer.js

```diff
@@ -231,7 +231,7 @@ module.exports = {
       data.name = data.name.trim()
     }
     ensureValidName(data.name, strict, options.allowLegacyCase)
-    if (isBuiltinModule(data.name)) {
+    if (moduleBuiltin.builtinModules.includes(data.name)) {
```

Comment:
Was fixing this today before noticing this PR.

A note here, if we want to ensure compatibility as before we should do something like this instead:
```suggestion
    if (moduleBuiltin.builtinModules.includes(data.name.replace(/^node:/, ''))) {
```

So `node:*` is handled.